stonesoup/reader/astronomical.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+"""Astronomical readers for Stone Soup.
```

Comment:
Astronomical readers don't seem to appear in the docs generated

Reply:
Oversight on my part. Added to latest commit.